inc/oapv.h

```diff
@@ -36,18 +36,16 @@
 extern "C" {
 #endif
 
-/* Not available on Android due to different build system */
-#ifndef __ANDROID__
+#if defined(OAPV_CONFIG_HEADER)
```

Comment:
Won't this lead to the header never getting included when installed, due to that define not being set anywhere but in the libraries CMakeLists?